src/main/java/teammates/ui/webapi/action/DeleteCourseAction.java

```diff
@@ -1,17 +1,15 @@
 package teammates.ui.webapi.action;
 
-import org.apache.http.HttpStatus;
-
-import teammates.common.exception.EntityDoesNotExistException;
-import teammates.common.exception.InvalidParametersException;
 import teammates.common.exception.UnauthorizedAccessException;
 import teammates.common.util.Const;
 
 /**
- * Action: Move a course to Recycle Bin (soft-delete) for an instructor.
+ * Action: Permanently deletes a course from Recycle Bin for an instructor.
  */
 public class DeleteCourseAction extends Action {
 
+    private String idOfCourseToDelete;
```

Comment:
Do you really need to do this?

Reply:
Um this was just a rename from DeleteInstructorSoftDeletedCourseAction > DeleteCourseAction. DeleteCourseAction used to be soft delete hence the changes. 
...
but yes it appears unnecessary. I'll remove it. IntelliiJ reports similar code in `RestoreInstructorSoftDeleteCourseAction` so that probably could be deleted as well.